src/theory/quantifiers/sygus_inst.h

```diff
@@ -124,6 +127,15 @@ class SygusInst : public QuantifiersModule
   /* Indicates whether a counterexample lemma was added for a quantified
    * formula in the current context. */
   context::CDHashSet<Node, NodeHashFunction> d_ce_lemma_added;
+
+  /* Set of global ground terms in assertions (outside of quantifiers). */
+  std::unordered_map<TypeNode,
```

Comment:
This also needs to be user-context dependent.

Reply:
done